CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 7.3.1
 
+-   Fixed: `number-*` now ignore values in comments and strings.
```

Comment:
This change can be removed. We are experimenting with leaving changelog nodes out of PRs, to reduce merge conflicts.